web/modules/custom/hoeringsportal_deskpro/composer.json

```diff
@@ -12,7 +12,7 @@
   },
   "require": {
     "deskpro/deskpro-api-client-php": "^2.0",
-    "drupal/advancedqueue": "^1.0",
+    "drupal/advancedqueue": "1.0.0-rc3",
```

Comment:
`^1.0` allows installation of `1.0.0-rc3` (as I understand it) so we should not change this.

Reply:
^1.0 also allows for installation of 1.0.0-rc7 which will cause https://www.drupal.org/files/issues/2021-10-20/add-getjobs-method-database-backend.patch patch to fail. It looks like a patch we need, so until there is an available patch in the https://www.drupal.org/project/advancedqueue/issues/3244940 thread or we update our code to not use getJobs() method we are stuck at rc3

Reply:
We should do something about out use of `getJobs`: https://github.com/itk-dev/hoeringsportal/blob/master/web/modules/custom/hoeringsportal_deskpro/src/Service/HearingHelper.php#L374-L378

Reply:
Moved getJobs method to hoeringsportalHelper class as a private method. Updated advanced queue module.